docs/faq.md

```diff
@@ -17,6 +17,10 @@ Backup Utilities as part of your disaster recovery plan.
 Backup Utilities are a disaster recovery tool. This tool takes date-stamped
 snapshots of all major datastores. These snapshots are used to restore an instance
 to a prior state or set up a new instance without having another always-on GitHub
-Enterprise instance (like the High Availability replica).
+Enterprise instance (like the High Availability replica). 
+
+Backup Utilities cannot be used on a GitHub Enterprise instance running as a High
+Availability replica. If you promote your High Availability replica to become the
+primary instance, you will need to add Backup Utilities at that time.
```

Comment:
Thanks for the PR.  I think this addition is best suited in a more prominent place like point 2 of "Getting started".

Something like the following change (some of the wording my need tweaking):

```diff
     backup and restore GitHub Enterprise 2.10 and earlier.

  2. Copy the [`backup.config-example`][3] file to `backup.config` and modify as
-    necessary. The `GHE_HOSTNAME` value must be set to the GitHub Enterprise
+    necessary. The `GHE_HOSTNAME` value must be set to the primary GitHub Enterprise
     host name. Additional options are available and documented in the
     configuration file but none are required for basic backup functionality.

+    As the data on a High Availability replica may be in a transient state at the time of backup,
+    Backup Utilities should not be used to backup data from a High Availability replica.
+
     * Backup Utilities will attempt to load the backup configuration from the following
       locations, in this order:
```

I've deliberately left off your last sentence as this shouldn't be needed as you'd still be performing the backup using the same hostname after failover.

Thoughts on wording @github/backup-utils ?